slatedb/src/compactor.rs

```diff
@@ -344,6 +418,8 @@ impl CompactorEventHandler {
             compaction_ts: db_state.last_l0_clock_tick,
             is_dest_last_run,
         };
+        self.progress_logger
+            .add_job(id, job.estimated_source_bytes());
```

Comment:
I opted to put this here instead of submit_job because we have access to the `CompactionJob` now, which can more easily calculate estimated size (I think).